linter.py

```diff
@@ -34,37 +31,24 @@ class Clang(Linter):
     multiline = True
 
     defaults = {
```

Comment:
I have another pull request for syntax-specific options #1 . It's a required feature for me personally for projects with both c and c++ source files (which is why I'm using SublimeLinter-gcc at the moment). What do you think about adding two defaults `"cflags"` and `"cxxflags"` like so:

    defaults = {
        '-Wall': True,
        '-I +': [],
        '-x': None,
        'cflags': [],
        'cxxflags': []
    }

Then check syntax and append the appropriate setting.